test/Microsoft.AspNetCore.Razor.Test/project.json

```diff
@@ -3,6 +3,7 @@
   "dependencies": {
     "dotnet-test-xunit": "2.2.0-*",
     "Microsoft.AspNetCore.Razor": "1.1.0-*",
+    "Microsoft.AspNetCore.Razor.Runtime": "1.1.0-*",
```

Comment:
Ya this sucks. Not a lot we can do though unless we made another runtime test sources package. Not sure that's worth it since it'd only have a single class.